src/components/SignOut.tsx

```diff
@@ -23,7 +20,21 @@ export const SignOut: React.FC<SignOutProps> = ({
   }
   return (
     <>
-      <DialogDisclosure as={Button} color={color} store={dialog}>
+      <DialogDisclosure
+        store={dialog}
+        sx={{
+          color: color,
+          cursor: "pointer",
+          background: "none",
+          border: "none",
+          padding: 0,
+          font: "inherit",
+          textDecoration: "underline",
+          "&:hover": {
+            opacity: 0.8
+          }
+        }}
```

Comment:
We could also match the style of the "{catalog} Home" and "My Books" button by importing `styleProps` from `Button/styles` instead of rendering this to look like a link:
```suggestion
        sx={styleProps(color, "md", "ghost")}
```

Reply:
Yes, much better! Thanks!